src/engraving/style/styledef.cpp

```diff
@@ -921,6 +921,8 @@ const std::array<StyleDef::StyleValue, size_t(Sid::STYLES)> StyleDef::styleValue
     { Sid::expressionAlign,               "expressionAlign",              Align(AlignH::LEFT, AlignV::BASELINE) },
     { Sid::expressionPlacement,           "expressionPlacement",          PlacementV::BELOW },
     { Sid::expressionOffset,              "expressionOffset",             PointF(.0, 2.5) },
+    { Sid::expressionPosAbove,            "expressionPosAbove",           PointF(.0, -1.5) },
+    { Sid::expressionPosAbove,            "expressionPosAbove",           PointF(.0, 2.5) },
```

Comment:
Shouldn't this be expressionPosBelow?

Reply:
urghhhh mistake, thank for spotting it 👍